UCSLibraries/pom.xml

```diff
@@ -83,6 +83,19 @@
 			<artifactId>wiremock</artifactId>
 			<version>2.18.0</version>
 		</dependency>
+		<dependency>
+		  <groupId>pl.pojo</groupId>
+		  <artifactId>pojo-tester</artifactId>
+		  <version>0.7.6</version>
+          <scope>test</scope>		  
```

Comment:
shouldn't the IDE be doing the automatic indentation on save?

Reply:
I think not. Only java classes gets formatted